### PR TITLE
Fix: Cloudtest sometimes does not save shell script's stdout/stderr

### PR DESCRIPTION
**Pull request by denis-tingajkin**
## Description
In the investigation of failing https://circleci.com/gh/networkservicemesh/networkservicemesh/105477 I found that sometimes cloudtest shell script result can be not saved.  
Root cause: we using async read from stdout/stderr and in the case when the command exits we can lose the output.

## Motivation and Context
https://circleci.com/gh/networkservicemesh/networkservicemesh/105477

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [X] Added unit testing to cover
- [X] Tested locally
- [ ] Have not tested

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: test/cloudtest/pkg/utils/shell_utils_test.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"strings"
+	"testing"
+
+	"github.com/onsi/gomega"
+)
+
+func TestProcessOutputShouldNotLostOutput(t *testing.T) {
+	assert := gomega.NewWithT(t)
+	const expected = "output..."
+	for i := 0; i < 1000; i++ {
```
Comment (haiodo) [inline test/cloudtest/pkg/utils/shell_utils_test.go]: How long it take to execute this kind of test?

Comment (denis-tingajkin) [reply test/cloudtest/pkg/utils/shell_utils_test.go]: ~3s
`<testcase classname="github.com/networkservicemesh/networkservicemesh/test/cloudtest/pkg/utils" name="TestProcessOutputShouldNotLostOutput" time="2.630000"/>` from JunitResult

Comment (haiodo) [reply test/cloudtest/pkg/utils/shell_utils_test.go]: I think it is not required to eat almost 3 seconds for such not important case.

Comment (denis-tingajkin) [reply test/cloudtest/pkg/utils/shell_utils_test.go]: @haiodo What the limit time should be per unit test?

Comment (denis-tingajkin) [reply test/cloudtest/pkg/utils/shell_utils_test.go]: finally reduced time to 1s

**Pull request merged**